server/form-pages/apply/move-on/pduRegion.ts

```diff
@@ -0,0 +1,46 @@
+import type { TaskListErrors } from '@approved-premises/ui'
+import { validPostcode } from '../../../utils/formUtils'
+
+import TasklistPage from '../../tasklistPage'
+
+export default class PduRegion implements TasklistPage {
+  name = 'pdu-region'
+
+  title = 'Placement duration and move on'
+
+  question = 'Probation delivery unit (PDU) postcode'
+
+  body: {
+    postcode: string
+  }
+
+  constructor(body: Record<string, unknown>) {
+    this.body = { postcode: body.postcode as string }
+  }
+
+  previous() {
+    return ''
+  }
+
+  next() {
+    return ''
+  }
+
+  response() {
+    return {
+      [this.question]: this.body.postcode,
+    }
+  }
+
+  errors() {
+    const errors: TaskListErrors<this> = {}
+
+    if (!this.body.postcode) {
+      errors.postcode = 'You must enter a postcode for the PDU location'
```

Comment:
I was thinking we would just ask for the first part of the postcode here, as exact postcodes might be too granular. Also, I was under the impression that we'd be asking for the the postcode area of where they're relocating to, not necessarily the PDU region

Reply:
Ok yeah that makes sense, I think I misread the question.
I've asked Selina for some content input on this because I think we could do with it and we can come back to it to improve later on but I will clarify the error for now